R/pipe_continuation_linter.R

```diff
@@ -0,0 +1,41 @@
+#' @describeIn linters check that all commas are followed by spaces, but do not
+#' have spaces before them.
+#' @importFrom xml2 xml_find_all as_list
+#' @export
+pipe_continuation_linter <- function(source_file) {
```

Comment:
I think `%>%` is one operator amongst many. It would make sense to lint it in `infix_spaces_linter()`. In this instance, space is to be interpreted loosely as "\s" or "\n".